assets/templates/header.inc.html

```diff
@@ -17,7 +17,14 @@
                     <li class="source_header_nav_i"><a class="source_a_l" href="#777">Link</a></li>
                 </ul>
             </nav>
+            <div class="sjs-login"></div>
```

Comment:
Don't edit the header, add login hook dynamicaly. Header is overriden from user folder.